### PR TITLE
Make InstanceGraph track module hierarchies not contained in the top-…

### PR DESCRIPTION
**Pull request by albert-magyar**
…level hierarchy

### CODE REVIEW

File: src/main/scala/firrtl/analyses/Netlist.scala [hunk 1]
```diff
@@ -56,7 +63,8 @@ class InstanceGraph(c: Circuit) {
   /** A list of absolute paths (each represented by a Seq of instances)
     * of all module instances in the Circuit.
     */
-  lazy val fullHierarchy = graph.pathsInDAG(topInstance)
+  val trueTopInstance = WDefInstance(c.main, c.main)
```
Comment (edwardcwang) [inline src/main/scala/firrtl/analyses/Netlist.scala]: Should the comment be attached to fullHierarchy?

**Comment by albert-magyar** [comment]
I moved the comment. Have you tried this with your constprop input?

**Comment by edwardcwang** [comment]
Yep, was in the process of testing it.

**Comment by edwardcwang** [comment]
Seems to work fine. Thanks Albert!

**Pull request merged**